lametro/templatetags/lametro_extras.py

```diff
@@ -327,7 +332,7 @@ def sort_topics(topics):
 
 @register.simple_tag
 def get_alerts():
-    return Alert.objects.all()
+    return Alert.objects.exclude(expiration__lt=timezone.now())
```

Comment:
The actual date filtering

Reply:
Hm, if dates are stored in the DB in UTC, should we pass a local time or a UTC time for filtering?

Reply:
UTC for sure! And `timezone.now()` operates in UTC in this case.

At time of writing, `timezone.now` returns this in the review app's shell:
`datetime.datetime(2025, 3, 13, 14, 13, 32, 744054, tzinfo=<UTC>)`

And the current time in UTC is 2:13 PM, so this lines up.